pkg/importer/dataStream.go

```diff
@@ -899,8 +909,9 @@ func (d *DataStream) copy(dest string) error {
 	if util.GetAvailableSpace(d.ScratchDataDir) > int64(0) {
 		defer CleanDir(d.ScratchDataDir)
 	}
-	if d.isHTTPQcow2() {
-		err := d.convertQcow2ToRawStream(dest)
+	var err error
+	if d.isHTTPQcow2() || d.isIsoImage {
+		err = d.streamFileToDest(dest)
```

Comment:
I think it would be cleaner to have separate functions for the `isHTTPQcow2()` case and the `isIsoImage` case rather than having `streamFileToDest` handle both